scripts/post_jenkins_result.py

```diff
@@ -1,53 +1,52 @@
 #!/usr/bin/env python2
 import json
-import os
 import os.path
 from datetime import datetime
 
+import requests
+
 from artifactor.plugins.post_result import test_report
-from cfme.utils import read_env
-from cfme.utils.path import project_path
+from cfme.utils.conf import cfme_data, credentials
 from cfme.utils.trackerbot import post_jenkins_result
 
 job_name = os.environ['JOB_NAME']
 number = int(os.environ['BUILD_NUMBER'])
 date = str(datetime.now())
 
-# reduce returns to bools for easy logic
-runner_src = read_env(project_path.join('.jenkins_runner_result'))
-runner_return = runner_src.get('RUNNER_RETURN', '1') == '0'
-test_return = runner_src.get('TEST_RETURN', '1') == '0'
+# no env var for build status, have to query Jenkins API directly and parse json
+jenkins_url = cfme_data.jenkins.url
+jenkins_user = credentials.get(cfme_data.jenkins.credentials).msgbus_username
+jenkins_token = credentials.get(cfme_data.jenkins.credentials).msgbus_token
+build_data_url = '/'.join([jenkins_url, 'job', job_name, 'lastBuild', 'api', 'json'])
+build_data = requests.get(build_data_url,
+                          verify=False,
+                          auth=(jenkins_user, jenkins_token))
+if build_data.status_code != 200:
+    raise ValueError('Bad return status ({}) from jenkins lastBuild API url: {}'
+                     .format(build_data.status_code, build_data_url))
+else:
+    build_data_json = build_data.json()
 
+build_status = build_data_json.get('result')
 
-# 'stream' environ is set by jenkins for all stream test jobs
-# but not in the template tester
-if job_name not in ('template-tester', 'template-tester-openstack',
-                    'template-tester-rhevm', 'template-tester-virtualcenter'):
-    # try to pull out the appliance template name
-    template_src = read_env(project_path.join('.appliance_template'))
-    template = template_src.get('appliance_template', 'Unknown')
-    stream = os.environ['stream']
-else:
-    tester_src = read_env(project_path.join('.template_tester'))
-    stream = tester_src['stream']
-    template = tester_src['appliance_template']
+stream = os.environ['stream']
+template = os.environ['appliance_template']
 
 if test_report.check():
     with test_report.open() as f:
         artifact_report = json.load(f)
 else:
-    raise RuntimeError('Unable to post to jenkins without test report: '
-        '{} does not exist!'.format(test_report.strpath))
+    raise RuntimeError('Unable to post to jenkins without test report: {} does not exist!'
+                       .format(test_report.strpath))
 
-if runner_return and test_return:
-    build_status = 'success'
-elif runner_return:
-    build_status = 'unstable'
-else:
-    build_status = 'failed'
 
-result_attrs = ('job_name', 'number', 'stream', 'date', 'template',
-    'build_status', 'artifact_report')
+result_attrs = ('job_name',
```

Comment:
**REQUIRED** remove the quotes and the eval calls

Reply:
I didn't write it... but okay I'll remove it, easy enough :)